proofs/lfsc/signatures/strings_programs.plf

```diff
@@ -1,11 +1,20 @@
 ; depends: nary_programs.plf theory_def.plf
 
-; Make empty string of the given string-like sort s.
-(program sc_mk_emptystr ((s sort)) term
-  (match s
-    ((Seq t) (seq.empty s))
+; This signature is used for both strings and sequences, where we often
+; write "string" in the documentation to refer to a string or sequence.
+
+; Make empty string of the given string-like sort u.
+(program sc_mk_emptystr ((u sort)) term
+  (match u
+    ((Seq t) (seq.empty u))
     (String emptystr)))
 
+(program sc_string_is_empty ((s term)) flag
```

Comment:
maybe add some documentation to this function? 
Same for other undocumented functions.